radar-spring-auth/src/main/kotlin/radar/spring/auth/managementportal/ManagementPortalAuthorization.kt

```diff
@@ -29,21 +29,23 @@ class ManagementPortalAuthorization() : Authorization<RadarToken> {
         source: String?
     ): Boolean {
         return runBlocking {
+            val subject = user.takeIf { it != null } ?: token.subject
```

Comment:
can you just use `user ?: token.subject` ?

Reply:
Made the changes. I also missed the public key endpoints bit. I've added it now.